### DISCUSSION TITLE
Passive scalar transport in THM

### DISCUSSION THREAD

**Discussion by tannhorn** [idaholab/moose#21750] [Q&A Modules: General] [answered]
Hello, I wanted to ask, is there a straightforward way to add passive scalar transport into a THM run? Or would it be necessary to rewrite the components so that they support an additional equation to be solved? I couldn't find any answer in the docs/reading through the source code.

**Comment by GiudGiud** [comment, marked as answer]
@licharlot

**Reply by licharlot** [reply]
THM doesn't support passive scalar. The additional equation for the passive scalar transport would have to be added to the flow model.

**Reply by lindsayad** [reply]
We are planning to add a scalar transport module very soon that would build off of Navier-Stokes and THM modules

**Reply by tannhorn** [reply]
Thank you for the replies!
